src/app/layout.tsx

```diff
@@ -2,35 +2,33 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { ReactNode } from 'react'
-import { Inter } from 'next/font/google'
-import './globals.css'
-import { getLocale } from './functions/server/l10n'
-import { getServerSession } from 'next-auth'
-import { SessionProvider } from '../contextProviders/session'
+import { ReactNode } from "react";
+import { Inter } from "next/font/google";
+import "./globals.css";
+import { getLocale } from "./functions/server/l10n";
+import { SessionProvider } from "../contextProviders/session";
+import { getCurrentSession } from "./functions/server/sessionHelpers";
 
-const inter = Inter({ subsets: ['latin'] })
+const inter = Inter({ subsets: ["latin"] });
 
 export const metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app'
-}
+  title: "Create Next App",
+  description: "Generated by create next app",
```

Comment:
Are these user facing at all? (ie: getting somehow shimmed into `<meta>` tags?)
They look a bit suspicious.

Reply:
I believe these are intended for meta tags so they will show up in social media and search engines and we do indeed want to set them appropriately/

Reply:
OK, think I found some useful docs on this&hellip; https://nextjs.org/docs/app/api-reference/functions/generate-metadata 👀

Reply:
Yes, these are user-facing. We’ll have to port over the meta tags from the current Monitor site for which we have a separate task. :)